kbase-extension/static/kbase/js/common/ui.js

```diff
@@ -40,6 +40,320 @@ define([
             .replace(/>/, '&gt;');
     }
 
+    /**
```

Comment:
There are a number of functions in this file that do not require the instantiation of a UI instance, so I moved them out so they are available as both `UI.<function>` and `uiInstance.<function>`.